.travis.yml

```diff
@@ -148,7 +148,7 @@ script:
     - popd
     # test that the embedding code works on our installation
     - mkdir /tmp/embedding-test &&
-        make check -C /tmp/julia/share/doc/julia/examples/embedding \
+        make check -C embedding \
```

Comment:
Why did you get rid of the path here?  I don't think this works; you still need to give it the path, otherwise it can't find it.

Reply:
Yeah I don't know why I did that. :upside_down_face: